packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/AmplifyApiPlugin.kt

```diff
@@ -40,13 +42,48 @@ class AmplifyApiPlugin : FlutterPlugin, MethodCallHandler {
         LOG.info("Initiated API plugin")
     }
 
-    override fun onMethodCall(@NonNull call: MethodCall, @NonNull result: Result) {
-        when (call.method) {
-            "query" ->
-                FlutterGraphQLApi.query(result, call.arguments as Map<String, Any>)
-            "mutate" ->
-                FlutterGraphQLApi.mutate(result, call.arguments as Map<String, Any>)
-            else -> result.notImplemented()
+    override fun onMethodCall(call: MethodCall, result: Result) {
+        var methodName = call.method
+
+        if(methodName == "cancel"){
+            onCancel(result, (call.arguments as String))
+            return
+        }
+
+        try {
+            var arguments : Map<String, Any> = call.arguments as Map<String,Any>
+
+            when (call.method) {
+                "get" -> FlutterRestApi.onGet(result, arguments)
+                "post" -> FlutterRestApi.onPost(result, arguments)
+                "put" -> FlutterRestApi.onPut(result, arguments)
+                "delete" -> FlutterRestApi.onDelete(result, arguments)
+                "head" -> FlutterRestApi.onHead(result, arguments)
+                "patch" -> FlutterRestApi.onPatch(result, arguments)
```

Comment:
Can we drop the prefix 'on' and just use the API name directly:
eg. `FlutterRestApi.get()` similar to how we have `FlutterGraphQLApi.query()`